README.md

```diff
@@ -1,46 +1,28 @@
 # highlightjs-curl
 
-Support for using `highlight.js` to syntax highlight cURL commands. See https://highlightjs.org/ for more information about highlight.js.
-
-See https://curl.haxx.se/docs/manpage.html or in your shell use `curl --help` for more information about cURL.
+Support for using `highlight.js` to syntax highlight cURL commands. See https://highlightjs.org/ for more information about highlight.js. See https://curl.haxx.se/docs/manpage.html or in your shell use `curl --help` for more information about cURL.
 
 ## Usage
 
-Include the `highlight.js` script package in your webpage or node app, load this module and register it with `hljs`. Follow instructions at [highlightjs](https://highlightjs.org/) to learn how to include the library and CSS.
-
-If you're not using a build system and just want to embed this in your webpage:
-
-```html
-<script type="text/javascript" src="/path/to/highlight.pack.js"></script>
-<script type="text/javascript" src="/path/to/highlightjs-curl/curl.js"></script>
-<script type="text/javascript">
-    hljs.registerLanguage('curl', window.hljsDefineCurl);
-    hljs.initHighlightingOnLoad();
-</script>
-```
+Include the `highlight.js` script package in your webpage or node app, load this module and register it with `hljs`. Follow instructions at [highlightjs](https://highlightjs.org/usage/) to learn how to include the library and CSS. See [Getting started](https://github.com/highlightjs/highlight.js#getting-started) for different integration and module options.
```

Comment:
I still don't think this is an improvement (removing the example).  I'm pretty sure our template repo when finished will have a complete example - vs making them go look elsewhere to see how all the pieces go together.